homu/main.py

```diff
@@ -439,9 +444,21 @@ def fetch_mergeability(mergeable_que):
         finally:
             mergeable_que.task_done()
 
+def arguments():
+    parser = argparse.ArgumentParser(description = 'A bot that integrates with GitHub and '
+                                     + 'your favorite continuous integration service')
+    parser.add_argument('-v', '--verbose',
+                        action='store_true', help='Enable more verbose logging')
+
+    return parser
+
 def main():
+    args = arguments().parse_args()
+
     logger = logging.getLogger('homu')
-    logger.setLevel(logging.INFO)
+    level = logging.DEBUG if args.verbose else logging.INFO
+    logger.setLevel(level)
+
```

Comment:
How about inlining these 3 lines, eliminating the need of the `level` variable? It's purely my preference, so it is up to you!